lib/puppet-lint/plugins/check_hiera.rb

```diff
@@ -49,7 +49,7 @@
     # metrics
     'govuk_logging::logstream::metrics_only',
 
-    # Allow sharing of asset_manager_uploaded_assets_routes array between assets-origin and static
+    # Allow sharing of asset_manager_uploaded_assets_routes array between assets-origin and draft-assets
```

Comment:
Minor: This is more of a correction than a clarification, isn't it?

Reply:
I've updated the commit note to describe it as a correction instead of a clarification.